src/components/Tabs/TabInfo/TabInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Tab } from '../../../types/Tab';
+
+type Props = {
+  tab: Tab;
+  onTabSelected: (tab: Tab) => void;
```

Comment:
this function does not take a value
![image](https://user-images.githubusercontent.com/91411692/236445399-37261bb4-a23c-42bd-9bcd-df94a286d9df.png)